Sources/HAP/Server/Server.swift

```diff
@@ -91,6 +91,16 @@ public class Server: NSObject, NetServiceDelegate {
                         break
                     }
 
+                    // Fix to allow Bridges with spaces in name
+                    // HomeKit POSTs contain a hostname with \\032 replacing the space
+                    // which causes Kitura httpParser to baulk on the resulting URL.
+                    // Replacing '\\032' with '-' in the hostname allows Kitura to create a valid URL,
+                    // and the value is not used elsewhere.
+                    //
+                    if let hostname = request.headers["Host"]?[0], (hostname.contains("\\032")) {
+                        request.headers["Host"]![0] = hostname.replacingOccurrences(of:"\\032", with:"-")
```

Comment:
I don't think we can easily add a test for this? We should –probably separately from this bugfix– add some test infrastructure to test the basic http server edge cases.